### PR TITLE
feat: identify LOC with UUID.

### PR DESCRIPTION
**Pull request by gdethier**
logion-network/logion-internal#174

### CODE REVIEW

File: pallets/logion_loc/src/uuid.rs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+	pub fn bytes(self) -> [u8; 32] {
+		self.bytes
+	}
+}
+
+impl Default for UUID {
+	fn default() -> UUID {
+		UUID {
+			bytes: [42; 32]
```
Comment (benoitdevos) [inline pallets/logion_loc/src/uuid.rs:33]: Why 42 as a default value ? It's not in the range 0-9a-f ?

File: pallets/logion_loc/src/tests.rs [hunk 2]
```diff
@@ -1,11 +1,37 @@
...
+	let expected_code = [42; 32];
+	uuid.using_encoded(|ref slice| {
+		assert_eq!(slice, &expected_code);
+	});
+}
+
+#[test]
+fn it_decodes_uuid() {
+	let expected_uuid: UUID = UUID::new(vec![41; 32]).ok().unwrap();
+	let mut code: &[u8] = &[41; 32];
```
Comment (benoitdevos) [inline pallets/logion_loc/src/tests.rs:25]: 41 is not a valid value - strange choice for a test.

**Review by benoitdevos — COMMENTED** [review]
I would like some explanations - I feel like I miss something...

**Comment by gdethier** [comment]
Nevermind, for some reason (lack of sleep I guess... -_-') I was taking the false hypothesis that a UUID was holding on 32 bytes, but that's only for the string representation (without the dashes). The internal representation of a UUID is only 16 bytes wide and thus holds on a u128, which is already handled by current implementation. I am therefore closing the PR as the job will be done frontend-side.

**Pull request closed without merging**